services/queue/schemas/v1/task-group-response.yml

```diff
@@ -0,0 +1,31 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Task-Group Response"
+description: |
+  Response containing information about a task group.
+type:               object
+properties:
```

Comment:
Would it make sense to include schedulerId (and soon projectId) here as well, since they're already easily accessible?

Reply:
yeah! It is already included in the bottom. We'll add projectId later